at-hoangtran/Week3/Day4/Exercise3-Day4.rb

```diff
@@ -0,0 +1,44 @@
+class HtmlToText
+  def get_html(html)
+    stack = []
+    check = false
+    @result = ''
+    @arr = html.split('')
+    i = 0
+    while i < @arr.length
+      if @arr[i..(i + 4)].join == '<div>'
+        stack.push('div')
+        if check == true
+          @result << "\n"
+          check == false
+        end
+        i += 5
+      elsif @arr[i..(i + 3)].join == '<br>'
+        if check == true
+          @result << "\n"
+          check == false
+        end
+        i += 4
+      elsif @arr[i..(i + 5)].join == '</div>'
+        if (stack.pop == 'div') && (check == true)
+          @result << "\n"
+          check = false
+        end
+        i += 6
+      else
+        @result << @arr[i]
+        check = true
+        i += 1
+      end
+
+    end
+  end
+
+  def show
+    puts @result
+  end
+ end
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/b841bab6cd381fd2aea25772a4ea14c33b24929d/at-hoangtran/Week3/Day4/Exercise3-Day4.rb#L40"></span>Layout/EndAlignment: <code>end</code> at 40, 1 is not aligned with <code>class</code> at 1, 0.</td>
    </tr>
  </tbody>
</table>